pkg/api/plugin.go

```diff
@@ -14,6 +14,34 @@ const (
 	ContextKeyTenantID     ContextKey = "TenantID"
 )
 
+type PluginStep string
+
+const (
+	PluginStepDeploy                  PluginStep = "Deploy"
+	PluginStepInitialize              PluginStep = "InitializeCluster"
+	PluginStepUpdateDrain             PluginStep = "UpdateDrain"
+	PluginStepUpdateMasterVMRotation  PluginStep = "UpdateMasterVMRotation"
+	PluginStepUpdateInfraVMRotation   PluginStep = "UpdateInfraVMRotation"
+	PluginStepUpdateComputeVMRotation PluginStep = "UpdateComputeVMRotation"
+	PluginStepWaitForHealthz          PluginStep = "WaitForHealthz"
+	PluginStepWaitForNodes            PluginStep = "WaitForNodes"
+	PluginStepHealthCheck             PluginStep = "HealthCheck"
```

Comment:
This is confusing when we also have WaitForHealthz. `ConsoleHealthz` or something along those lines.

Reply:
ack, I'll fix